development/build/static.js

```diff
@@ -151,20 +151,20 @@ function getCopyTargets(
       pattern: `*.css`,
       dest: ``,
     },
-    {
-      src: `./app/loading.html`,
-      dest: `loading.html`,
-    },
-    {
-      src: shouldIncludeSnow
-        ? `./node_modules/@lavamoat/snow/snow.prod.js`
-        : EMPTY_JS_FILE,
-      dest: `snow.js`,
-    },
-    {
-      src: shouldIncludeSnow ? `./app/scripts/use-snow.js` : EMPTY_JS_FILE,
-      dest: `use-snow.js`,
-    },
+    ...(shouldIncludeSnow
+      ? [
+          {
+            src: shouldIncludeSnow
+              ? `./node_modules/@lavamoat/snow/snow.prod.js`
+              : EMPTY_JS_FILE,
+            dest: `snow.js`,
+          },
+          {
+            src: `./app/scripts/use-snow.js`,
+            dest: `use-snow.js`,
+          },
+        ]
+      : []),
```

Comment:
TL;DR: the rest of the PR prevents the code from ever trying to load snow, so we don't need to copy an empty file anymore.